src/game/client/swarm/vgui/asw_hud_master.cpp

```diff
@@ -480,6 +486,17 @@ void CASW_Hud_Master::Paint( void )
 
 	if ( m_pLocalMarineResource )
 	{
+		// draw restricted borders for ultra-wide screen
+		if (rd_draw_restricted_borders.GetBool() && g_ultra_wide_screen && !pPlayer->IsObserver()) {
```

Comment:
spectating is determined in AS:RD by whether the player has a non-null SpectatingNPC